src/module/rules/rule-element/crit-spec.ts

```diff
@@ -68,6 +68,8 @@ class CritSpecRuleElement extends RuleElementPF2e<CritSpecRuleSchema> {
                 });
                 const bonusValue = ABP.isEnabled(this.actor)
                     ? ABP.getAttackPotency(this.actor.level)
+                    : weapon.isOfType("melee")
+                    ? weapon.linkedWeapon?.system.runes.potency ?? 0
```

Comment:
This looks suspicious but also if it causes a bug it'll be easy to notice so might as well!